shuffle/move/examples/stdlib/Nursery/sources/Compare.move

```diff
@@ -0,0 +1,74 @@
+/// Utilities for comparing Move values based on their representation in BCS.
+module Std::Compare {
```

Comment:
Why is it in the examples directory?

Reply:
This stdlib copy will be removed in the future. This right now serves as a stop gap to allow a relative reference ("../stdlib") in generated shuffle projects.